Classes/Public/TextKit/AztecVisualEditor.swift

```diff
@@ -46,6 +45,49 @@ public class AztecVisualEditor : NSObject
     }
 
 
+    // MARK: - Misc helpers
```

Comment:
I'm going to leave it as is for now. I expect `paragraph` is too specific and I'm betting there will be some other helper type methods related to lists that we can group there.  Better naming should become apparent down the road.